### ISSUE TITLE
How to train data with multi-class label

### ISSUE THREAD

**Issue by hwjwayne** [apache/incubator-mxnet#8933]
There are 600 classes in the image dataset, each image belongs to one or more classes. How can I train such dataset?

**Is it a correct choice:**
Present image label by 600*1 binary vector, which contains one or more "1".
Bulid network with 600 outputs, and activate them with LogisticRegressionOutput()  (BTW, Will mxnet set the loss function to logistic loss automatically?)

I obtain a 0.49% mAP on this dataset, by it's reported 36.1% in the paper. (Parameters are set the same as the paper)

**Comment by leleamol** [comment]
Proposed Labels: "Question","Discussion"

**Comment by sandeep-krishnamurthy** [comment]
Can you please ask usage related questions at - https://discuss.mxnet.io/
